src/sentry/testutils/cases.py

```diff
@@ -587,6 +587,10 @@ def invoke(self, *args):
 
 @pytest.mark.usefixtures("browser")
 class AcceptanceTestCase(TransactionTestCase):
+    from sentry.utils.pytest.selenium import Browser
```

Comment:
TIL You can import inside classes.

Reply:
Is importing here any different to just importing up the top? I'd assume this gets run on module load

Reply:
Yeah I think you're right. I wanted to avoid any weird dependency stuff, but yeah this just runs at module load 🤷‍♂️